### ISSUE TITLE
Feature Request: Add multi-gpu support please

### ISSUE THREAD

**Issue by sushmitxo** [cocktailpeanut/fluxgym#14]
**Add multi-gpu support please**

**Comment by bnewddt** [comment]
yes - apple silicon too, I can't run it on a m2 max with 64gb unified ram

**Comment by jeffhaskin** [comment]
Seconded. M2 Pro chip with 32gb shared (regularly uses up to 25gb vram). I know there's a pytorch for apple silicon, are m-series versions of the the other dependancies for this?
